packages/aws-cdk/test/diff.test.ts

```diff
@@ -336,7 +336,7 @@ describe('non-nested stacks', () => {
     expect(exitCode).toBe(0);
   });
 
-  test('diff does not check for stack existence when --no-changeset is passed', async () => {
+  test('diff does not check for stack existence when changeSet is false', async () => {
```

Comment:
```suggestion
  test('diff does not check for stack existence when --no-change-set is passed, async () => {
```

Reply:
Just a comment on this - I removed that line because we don't actually check the flags - we just call diff with preconfigured arguments which don't map to the actual flag names.